starknet_py/tests/e2e/client/prepare_net_for_gateway_test.py

```diff
@@ -67,4 +92,29 @@ async def prepare_net_for_tests(
         block_with_invoke_number=block_with_invoke_number,
         declare_transaction_hash=declare_result.transaction_hash,
         block_with_declare_number=block_with_declare_number,
+        deploy_account_transaction_hash=deploy_account_result.transaction_hash,
+        block_with_deploy_account_hash=block_with_deploy_account_hash,
+        block_with_deploy_account_number=block_with_deploy_account_number,
+    )
+
+
+async def _signed_deploy_account_transaction(
```

Comment:
Isn't it a duplication of the `deploy_account_transaction` fixture? Maybe there is a chance to reuse it?